src/applications/gi/reducers/calculator.js

```diff
@@ -228,7 +227,7 @@ export default function(state = INITIAL_STATE, action) {
 
       let beneficiaryZIPError;
 
-      if (!beneficiaryZIPRegExTester.exec(beneficiaryZIP)) {
+      if (beneficiaryZIP.length > 5) {
```

Comment:
Shouldn't this be `.length < 5` since the number input will prevent numbers > 5 in length?

Reply:
Hello Mottie, this PR is still in draft I'm just testing some different things out. As for this if statement I want to throw the error if the length is greater than 5.